### PR TITLE
New package: mysql++-3.2.4

### PR DESCRIPTION
**Pull request by DirectorX**

### CODE REVIEW

File: srcpkgs/mysql++/template [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Template file for 'mysql++'
+pkgname=mysql++
+version=3.2.4
+revision=1
+build_style=gnu-configure
+configure_args="--prefix=/usr"
```
Comment (pullmoll) [inline srcpkgs/mysql++/template]: This line is redundant. Remove it.

**Comment by DirectorX** [comment]
ping @pullmoll

**Comment by pullmoll** [comment]
LGTM. Can anyone of @void-linux/pkg-committers test if it (cross) builds and merge?

**Comment by Hoshpak** [comment]
Unfortunately it does not cross-compile, see https://travis-ci.org/void-linux/void-packages/jobs/414872983 . libmariadbclient-devel could be removed from hostmakedepends if the path is specified in configure_args `configure_args="--with-mysql-include=${XBPS_CROSS_BASE}/usr/include/mysql"`. However the configure script still sets `-L/usr/lib64 ` which causes linking to fail when cross-compiling.

**Comment by DirectorX** [comment]
@pullmoll

**Comment by Hoshpak** [comment]
sure, why not.

**Pull request merged**